test/rule-matches/scrollable-region-focusable-matches.js

```diff
@@ -1,24 +1,60 @@
-describe('scrollable-region-focusable-matches', function() {
+describe('scrollable-region-focusable-matches', function () {
   'use strict';
 
   var fixture = document.getElementById('fixture');
   var queryFixture = axe.testUtils.queryFixture;
   var shadowSupported = axe.testUtils.shadowSupport.v1;
   var rule = axe.utils.getRule('scrollable-region-focusable');
 
-  afterEach(function() {
+  afterEach(function () {
     fixture.innerHTML = '';
   });
 
-  it('returns false when element is not scrollable', function() {
+  it('returns false when element is not scrollable', function () {
     var target = queryFixture(
       '<section id="target">This element is not scrollable</section>'
     );
     var actual = rule.matches(target.actualNode, target);
     assert.isFalse(actual);
   });
 
-  it('returns false when element has no visible children', function() {
+  it('returns false when the scrollable content is disabled and not focusable', function () {
+    var target = queryFixture(
+      '<select id="target" size="2" name="select" disabled>' +
+        '<option value="First">First</option>' +
+        '<option value="Second">Second</option>' +
+        '<option value="Third">Third</option>' +
+        '</select>'
+    );
+    var actual = rule.matches(target.actualNode, target);
+    assert.isFalse(actual);
+  });
+
+  it('returns false when the scrollable content is disabled="true" and not focusable', function () {
```

Comment:
Should add a test for parent fieldset having `disabled` as well as `aria-disabled`

Reply:
If the parent element is disabled does that not mean all children are disabled by default?

Reply:
edit:  added test for this scenario: 

```html
  <fieldset disabled>
    <legend>Do you like receiving wrenches?</legend>
    <input
      type="checkbox"
      id="chbx"
      name="agree"
      value="Yes!"
      aria-disabled="false"
    />
    <label for="chbx">Well, yes of course!</label>
  </fieldset>
  ```